### DISCUSSION TITLE
Weird left/top margin even if all margins are removed.

### DISCUSSION THREAD

**Discussion by martinrotter** [cefsharp/CefSharp#4710] [General]
Hi. Check this.

![image](https://github.com/cefsharp/CefSharp/assets/1255302/7d4df5d8-489c-470a-a08f-889b26072f64)

Somehow, my ChromiumWebBrowser instances in WPF app always leave behind left/top margin (the green color is set to parent control backgroun).

The HTML/CSS code sets padding/margin for body element to 0, WPF element/parent has Padding and Margin set to zero. Yet, there is this weird margin visible. Any ideas?

When viewing the HTML content in Firefox, there are no margins as expected.

![Screen Shot 2024-02-08 at 21 04 15](https://github.com/cefsharp/CefSharp/assets/1255302/08c5cafe-5422-42cc-a6c4-0f2805cb1f67)

**Comment by amaitland** [comment]
You can use `DevTools` to debug html/css/javascript issues See https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging

> When viewing the HTML content in Firefox, there are no margins as expected.

You need to compare to a `Chromium` based browser at the same version for a meaningful comparison.
